packages/experiments/src/utilities/createSlot.tsx

```diff
@@ -0,0 +1,210 @@
+import * as React from 'react';
+import { memoizeFunction } from 'office-ui-fabric-react';
```

Comment:
`// TODO: import this from utilities instead of oufr`

Reply:
Don't want to explicitly add this dependency to experiments temporarily so I've added a TODO in the code to do when this file is moved to Foundation.